data_handler/handler_tools/data_parser/serializers.py

```diff
@@ -83,7 +83,101 @@ def validate_valid_numbers(cls, value: str, info: ValidationInfo) -> Decimal:
         Returns:
             Decimal: Converted decimal value.
         """
+
+        if not value.isdigit():
+            raise ValueError("%s field is not numeric" % info.field_name)
+        return Decimal(str(int(value, base=16)))
+
+class RepaymentEventData(BaseModel):
+    """
+    Data model representing a repayment event in the system.
+
+    Attributes:
+        repayer (str): The address or identifier of the individual or entity making the repayment.
+        beneficiary (str): The address or identifier of the individual or entity receiving the repayment.
+        token (str): The type or symbol of the token being used for the repayment.
+        raw_amount (str): The raw amount of the repayment as provided, before any conversions or calculations.
+        face_amount (str): The face amount of the repayment, representing the value after necessary conversions.
+    """
+    repayer: str
+    beneficiary: str
+    token: str
+    raw_amount: str
+    face_amount: str
+
+class RepaymentEventSerializer(BaseModel):
```

Comment:
why do we need this class if can have only `RepaymentEventData`?
Please redudant it and move all logic there to `RepaymentEventData`